docs/source/en/training/lora.md

```diff
@@ -402,4 +402,8 @@ Thanks to [@isidentical](https://github.com/isidentical) for helping us on integ
 ### Known limitations specific to the Kohya-styled LoRAs
 
 * SDXL LoRAs that have both the text encoders are currently leading to weird results. We're actively investigating the issue. 
-* When images don't looks similar to other UIs such ComfyUI, it can be beacause of multiple reasons as explained [here](https://github.com/huggingface/diffusers/pull/4287/#issuecomment-1655110736). 
\ No newline at end of file
+* When images don't looks similar to other UIs such ComfyUI, it can be beacause of multiple reasons as explained [here](https://github.com/huggingface/diffusers/pull/4287/#issuecomment-1655110736). 
+
+## Stable Diffusion XL
+
+We support fine-tuning of the UNet shipped in [Stable Diffusion XL](https://huggingface.co/papers/2307.01952) with DreamBooth and LoRA via the `train_text_to_image_lora_sdxl.py` script. Please refer to the docs [here](https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/README_sdxl.md). 
```

Comment:
I think it makes sense to add this section to the respective training sections of DreamBooth and text-to-image. That way the information remains easier to parse.

Reply:
In my experiments, it needs in [this section](https://github.com/okotaku/diffusers/blob/train_lora_sdxl/examples/text_to_image/train_text_to_image_lora_sdxl.py#L1162-L1187).